index.html

```diff
@@ -269,47 +318,103 @@ <h4>SvgRenderingTemplate</h4>
               <td>
-A CSS Level 3 Media Query that can be used to specify desired characteristics
-in the display device. If there are multiple render methods, the best match
-to the CSS Level 3 Media Query will be used by default. It is expected that
-an individual could override this preference through a manual process.
+An OPTIONAL human-readable [=string=] that provides a more involved description
+than `name` of when the particular rendering might be useful.
+              </td>
+            </tr>
+            <tr>
+              <td>disclosedProperty</td>
+              <td>
+An OPTIONAL [=list=] of [=string=] values that each conform to the
+[[[RFC6901]]] syntax that specifies which properties from the [=verifiable
+credential=] are exposed when using this specific render method. If
+`disclosedProperty` is not provided, the entire [=verifiable credential=] is
+presumed to be shared when the render method is used.
+              </td>
+            </tr>
+            <tr>
+              <td>template</td>
+              <td>
+An OPTIONAL [=string=] or [=ordered map|map=] that provides the template that
+will be used to perform the rendering. If the value is a [=string=], it MUST be
+a URL. If the value is a [=ordered map|map=], it MUST conform to the following
+rules:
+                <table class="simple">
+                  <thead>
+                    <tr>
+                      <th style="white-space: nowrap">Property</th>
+                      <th>Description</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td>id</td>
+                      <td>
+An REQUIRED [=string=] that follows the [[[URL]]] and, when fetched,
+dereferences to a template such as an SVG or PDF file.
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>mediaType</td>
+                      <td>
+A RECOMMENDED [=string=] that identifies the media type for the `id` value
+as specified in [[[RFC6838]]].
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>digestMultibase</td>
+                      <td>
+An OPTIONAL multibase-encoded Multihash of the template file. The multibase
+value MUST be `z` and the multihash value MUST be SHA-2 with 256-bits of output
+(`0x12`).
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
               </td>
             </tr>
             <tr>
               <td>digestMultibase</td>
                 <td>
-An optional multibase-encoded multihash of the SVG image. The multibase value
-MUST be `z` and the multihash value MUST be SHA-2 with 256-bits of output
-(`0x12`).
+An OPTIONAL multibase-encoded Multihash of the render method referenced if `id`
+is specified. The multibase value MUST be `z` and the multihash value MUST be
+SHA-2 with 256-bits of output (`0x12`).
```

Comment:
I think base64url-nopad should be used here (prefix `u`), however, I also think we should allow for canon-hashes to be used here so that, for example, JCS can be performed on the render method description prior to hashing. This is a big help with files that might be served with slightly differing whitespace, etc. over time that doesn't change the semantics.